secretservice/secretservice.go

```diff
@@ -0,0 +1,305 @@
+package secretservice
```

Comment:
We should probably use

    // +build linux

for this.

Reply:
You can use dbus on OSX if you want to, is it ok to leave it off? In the client this package will only be imported under `+build linux`

Reply:
Sure.